### PR TITLE
Switch from macs2 to macs3

### LINKED ISSUE #378: Update MACS to v3
### Description of feature

With the release of MACS version 3 could you implement the new version?

Thank you in advance!

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Closes #378

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -579,7 +579,7 @@ if (!params.skip_peak_annotation) {
             ext.args   = '-gid'
```
Comment (mirpedrol) [inline conf/modules.config:579]: The name in the previous line is not updated `BAM_PEAKS_CALL_QC_ANNOTATE_MACS2_HOMER:HOMER_ANNOTATEPEAKS`

File: test.csv
Comment (mirpedrol) [inline test.csv:1]: This was probably added by mistake? 🙂

**Review by mirpedrol — APPROVED** [review]
LGTM! only two small comments :)

File: conf/modules.config (hunk 1 above)
Comment (JoseEspinosa) [reply conf/modules.config:579]: Good spot, did not save the change 😓

**Pull request merged**